Source/core/JSONRPC.h

```diff
@@ -108,36 +108,36 @@ namespace Core {
                         Code = -32000; // Server defined, now mapped to Timed out
                         Text = _T("Call timed out.");
                         break;
-                    case Core::ERROR_PARSE_FAILURE:
+                    case Core::ERROR_PARSING_ENVELOPPE:
                         Code = -32700; // Parse error
                         Text = _T("Parsing of the parameters failed");
                         break;
                     case Core::ERROR_INVALID_RANGE:
-                        Code = Core::ERROR_INVALID_RANGE;
+                        Code = -33000 - Core::ERROR_INVALID_RANGE;
```

Comment:
Perhaps make a const(expr) out of the 33000 with a meaningful name so it is more clear why we use it as a base for the application errors? If you look in the json rpc spec (as I was curious why it was 33000 anyway :) ) it looks like the codes were written with a signed 16 bits code in mind, but if 32 bit can be used indeed then this should be correct as well.